### PR TITLE
Uploading centos7 spec file for postgress age rpm

### PR DESCRIPTION
**Pull request by khurramlatif91**
Added spec file to create postgress age rpm on centos7

### CODE REVIEW

File: centos7.spec [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Version:        1
+Release:        %{dist}
+License:        Apache
+Group:          Databases
+Url:            https://github.com/apache/age.git
+Source0:        %{name}-%{version}.tar.gz
+BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+
+%description
+This package provides extensions for Apache Age
```
Comment (uhayat) [inline centos7.spec:15]: Add a bit details in description like bellow:
Apache AGE is an extension for PostgreSQL that enables users to leverage a graph database on top of the existing relational databases.

File: centos7.spec [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+%install
+cp -p age.so %{pginstdir}/lib/age.so
+cp -p age--1.1.1.sql %{pginstdir}/share/extension/age--1.1.1.sql
+cp -p age.control %{pginstdir}/share/extension/age.control
+
+%files
+
+%changelog
+* Fri Feb 17 2023 - Khurram Latif <Khurram.L@agedb.io>
+- Initial packaging for AEP
```
Comment (uhayat) [inline centos7.spec:37]: Fix type, Use AGE instead of AEP

**Review by uhayat — CHANGES_REQUESTED** [review]

File: centos7.spec (hunk 2 above)
Comment (khurramlatif91) [reply centos7.spec]: Updated description and changelog comment as per your suggestion

**Review by uhayat — APPROVED** [review]
Looks good, ready to merge.

**Pull request merged**